graylog2-web-interface/src/components/content-packs/ContentPackInstallView.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Row, Col } from 'react-bootstrap';
+
+import 'components/content-packs/ContentPackDetails.css';
+
+const ContentPackInstallView = (props) => {
+  const { comment } = props.install;
+  const createdAt = props.install.created_at;
```

Comment:
You can shorten this by using destructuring:

```
const { comment, createdAt, createdBy } = props.install;
```

Reply:
Not exactly:

`const { comment, created_at, created_by } = props.install;`

But the would generate a bunch of warnings I would have to silent. Which I would not like to do.